src/server.ts

```diff
@@ -146,7 +146,7 @@ export class Server extends http.Server {
 
 }
 
-type V8ResponseBody = null | string | ArrayBuffer | Buffer | ivm.Reference<ProxyStream>
+type V8ResponseBody = null | string | Uint8Array | ArrayBuffer | Buffer | ivm.Reference<ProxyStream>
```

Comment:
That's probably fine, but we're not handling it in the `handleResponse`. This only compiles because `Writable.prototype.end` accepts the `any` type. I'm not sure how it would behave if it got an actual Uint8Array though.

Reply:
Okay, I will get rid of that.